pkgs/by-name/vl/vlc-bittorrent/package.nix

```
@@ -0,0 +1,50 @@
+{ lib
+, autoconf-archive
+, autoreconfHook
+, boost
+, fetchFromGitHub
+, libtorrent-rasterbar
+, libvlc
+, openssl
+, pkg-config
+, stdenv
+}:
+
+# VLC does not know where the vlc-bittorrent package is installed.
+# make sure to have something like:
+#   environment.variables.VLC_PLUGIN_PATH = "${pkgs.vlc-bittorent}";
+
+stdenv.mkDerivation rec {
+  pname = "vlc-bittorrent";
+  version = "2.15.0";
+
+  src = fetchFromGitHub {
+    owner = "johang";
+    repo = pname;
+    rev = "6810d479e6c1f64046d3b30efe78774b49d1c95b";
```

Any reason why not using 

```suggestion
    rev = "v${finalAttrs.version}";
```

?

---

I wasn't really thinking it through. I guess, technically, upstream *could* push a different commit with tag `v2.15.0`, but that is highly unlikely. This one is probably nicer to maintain.

---

This is really unlikely.  In the cases it happens, we prefer to comment with something like:

```nix
# upstream use tags as moving targets
```

To be honest I have seen it only once -  in the `tar.gz` file provided by Xorriso!

---

Yeah, after thinking about it for a couple minutes I realized how unlikely it is. Made the change.